docs/README.md

```diff
@@ -0,0 +1,57 @@
+# Set up HashBus for an event
+The purpose of this document is to describe how to set up HashBus to monitor tweets and display leaderboard.

```

Comment:
Yeah, I started with different intent ;) I will correct this sentence when we agree on the structure